Library/ViewModels/NoShippingPledgeViewModel.swift

```diff
@@ -1244,3 +1248,17 @@ private func pledgeAmountSummaryViewData(
     rewardIsLocalPickup: rewardIsLocalPickup
   )
 }
+
+private func mockPledgePaymentIncrement() -> [PledgePaymentIncrement] {
```

Comment:
Merging mock data into the main branch makes me nervous. It looks like it should be safe, since it's behind `featurePledgeOverTimeEnabled()`, but I'd worry about us making a mistake with the feature rollout and accidentally showing mock data to users on older versions of the app.

Can you put the mock code in a `#if DEBUG` macro block, or just comment it out when merging and return an empty array?

Reply:
Good point, Amy. I think using `#if DEBUG` is the best approach here to ensure the mock data doesn’t accidentally appear in production. I’ll update the code accordingly and push the changes.